pypeit/core/mosaic.py

```diff
@@ -0,0 +1,312 @@
+"""
+Provide basic mosaicing functions.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+from IPython import embed
+
+import numpy as np
+from scipy import ndimage
+
+from pypeit import msgs
```

Comment:
Is there a reason not to have the mosaic and transform functions all in one module?

Reply:
They're distinct in my mind.  `pypeit/core/transform.py` is for generating/applying coordinate transformations, whereas `pypeit/core/mosaic` is for remapping input images to a mosaic.  I'd prefer to keep them separate.